vendor/assets/bower_components/jquery/src/event/ajax.js

```diff
@@ -0,0 +1,13 @@
+define([
+	"../core",
+	"../event"
+], function( jQuery ) {
+
+// Attach a bunch of functions for handling common AJAX events
+jQuery.each( [ "ajaxStart", "ajaxStop", "ajaxComplete", "ajaxError", "ajaxSuccess", "ajaxSend" ], function( i, type ) {
```

Comment:
Missing "use strict" statement.<br>Expected 'jQuery' to have an indentation at 5 instead at 1.